src/lib/components/Instances/InstanceTable.svelte

```diff
@@ -103,6 +103,7 @@
 						project.show_labels = true;
 						const projectRes = await toggleProjectLabels(
 							project.id,
+							prompt.prompt,
```

Comment:
I'd send the whole prompt. We can infer from meta vars as well. E.g. if it's JSON we probably want to just compare json in a metric.

Reply:
good idea